src/gtimelog/shortcuts.ui

```diff
@@ -101,6 +101,13 @@
                 <property name="title" translatable="yes" context="shortcut window">Menu</property>
               </object>
             </child>
+            <child>
+              <object class="GtkShortcutsShortcut">
+                <property name="visible">True</property>
+                <property name="accelerator">&lt;primary&gt;L</property>
+                <property name="title" translatable="yes" context="shortcut window">Add new entry</property>
```

Comment:
Perhaps this should say "Focus the task entry"?  "Add" sort of implies this keybinding would trigger the Add button.